### DISCUSSION TITLE
Migrate local repository to remote

### DISCUSSION THREAD

**Discussion by ghubz** [WeblateOrg/weblate#7734] [General]
Hi,

Is it possible to migrate a project from a local repository to a new remote one? Or is it required to start over?

**Comment by nijel** [comment]
Yes, merge Weblate local repository into remote one and set repository URL in Weblate to it.

**Comment by ghubz** [comment]
Thanks.

Another quick one. Can you create multiple branches in the local repo for a component via the interface? Basically, does the interface handle only one branch at a time?

**Reply by nijel** [reply]
No, the local repo is always handling only single branch.

**Reply by ghubz** [reply]
Ok thanks. Does revision control work only with remote git? Not the local one?

**Reply by nijel** [reply]
The local one is the same as remote - it does the same commits, it only does not have remote and thus cannot push to it or pull from it.
